test/extended/util/image/image.go

```diff
@@ -11,6 +11,9 @@ import (
 
 func init() {
 	allowedImages = map[string]int{
+		// used by jenkins tests
+		"quay.io/redhat-developer/nfs-server:latest": -1,
```

Comment:
Any reason for `latest` here, any chance for more stable tag instead?

Reply:
Looks like this is the tag that has been used so far: https://github.com/openshift/origin/pull/26088/files#diff-33893020d11fb9139a6cd5292c42e041dc2c7e80a6fae76e28fef5386096d780L34

Reply:
fixed